rapids-cmake/cmake/build_type.cmake

```diff
@@ -21,13 +21,13 @@ rapids_cmake_build_type
 
 .. versionadded:: v21.06.00
 
-Establish the :cmake:variable:`CMAKE_BUILD_TYPE` default value.
+Establish the :cmake:variable:`CMAKE_BUILD_TYPE <cmake:variable:CMAKE_BUILD_TYPE>` default value.
```

Comment:
Can you remind me why the double reference is necessary again? I'm wondering if there is some way to update conf.py with some extra role-related information to help Intersphinx link these correctly.

Reply:
This may not be a helpful comment, but I looked into this at one point and I _think_ it had to do with a limitation in CMake's rst object inventory or some other thing we cannot control. I concluded it wasn't worth fighting and that we should just accept it, but it's possible that I overlooked some possible solutions.

Reply:
What @bdice says is my understanding as well. The CMake rst setup wasn't designed for external usage and we have to jump through some hoops to cross reference it.

Reply:
Got it, that's fine then.